scipy/sparse/linalg/dsolve/SuperLU/SRC/msc_stdint.h

```diff
@@ -33,6 +33,11 @@
 #error "Use this header only with Microsoft Visual C++ compilers!"
 #endif // _MSC_VER ]
 
+#if _MSC_VER >= 1900
```

Comment:
This probably should preferably be in the places that #include "msc_stdint.h", so that we retain this file as a verbatim copy of the msinttypes file.